TankGame.cs

```diff
@@ -41,8 +41,8 @@ public class GameConfig
         public static bool BorderlessWindow { get; set; } = true;
```

Comment:
why are the graphics settings still static even though the other settings are not?